### PR TITLE
Drag and Drop Question

### PR DESCRIPTION
**Pull request by moorscode**
Titles and Paragraphs ordering question
Has to be set hard in code (ID)

### CODE REVIEW

**Comment by moorscode** [comment]
Requires https://github.com/Yoast/Yoast-theme-public/pull/22 to be merged.

File: php/class-llms-quiz-answers-as-paragraph.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Yoast\YoastCom\Theme;
+
+class LLMS_Quiz_Answers_As_Paragraph {
+
+	/**
+	 * Alle opties achter elkaar als "paragraaf"
```
Comment (atimmer) [inline php/class-llms-quiz-answers-as-paragraph.php]: Wat is dit voor rare taal? 🤔

Comment (moorscode) [inline php/class-llms-quiz-answers-as-paragraph.php]: Sorry I only understand engrish.

File: php/class-llms-quiz-answers-as-paragraph.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		if ( is_admin() ) {
+			return;
+		}
+
+		add_filter( 'the_content', array( $this, 'filter_the_content' ) );
+
+		add_action( 'lifterlms_single_question_before_summary', array( $this, 'question_before_summary' ) );
+	}
+
+	public function question_before_summary( $args ) {
```
Comment (atimmer) [inline php/class-llms-quiz-answers-as-paragraph.php]: Empty function?

**Comment by atimmer** [comment]
CR Done 👍🏻

**Pull request merged**